README.md

```diff
@@ -1,82 +1,42 @@
 ![#badassfish](payara-logo-blue.png)
 
-# Payara Server 5
+# Payara Server 5 Community Edition
```

Comment:
Maybe we should remove the version number, what do you think?

Reply:
I'm happy to leave it.
On the other hand is there a reason we've got both _Payara Server 5 Community Edition_ and _Payara Platform 5 Community Edition_ headings?

Reply:
The reason is that I didn't find a better solution. The original title was Payara Server but the second half of the page was about Platform (features shared by Server and Micro). So I added a new title for the Platform part. If it's OK to change the original title from Payara Server to Payara Platform then it's enough to have a single Payara Platform title.

Reply:
Our docs at https://docs.payara.fish/ now say Payara Platform so it should be OK to change the main title to Payara Platform and remove the second title. Or add a separate title for the Payara Server section and also for the Payara Micro section. Then we would have the main title "Payara Platform" and 2 section titles "Payara Server" and "Payara Micro".